c_tests/test_Reader.cpp

```diff
@@ -9,6 +9,9 @@
 #include "../_aicspylibczi/Reader.h"
 #include "../_aicspylibczi/pb_helpers.h"
 
+
+#define CORES_FOR_THREADS 1 // for local testing increase this but for GH Actions it must be 1
```

Comment:
interesting, why doesn't GH Actions allow more than 1?

Reply:
Seconding: https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners

They have 2 cores.